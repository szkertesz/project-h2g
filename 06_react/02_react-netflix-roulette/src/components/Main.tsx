import Genres from "./Genres"
import MovieList from "./MovieList";

function Main() {
    return (
        <main>
            <Genres />
            <MovieList />
        </main>
    );
}

export default Main