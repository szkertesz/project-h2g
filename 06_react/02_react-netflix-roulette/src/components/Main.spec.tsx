import React from 'react';
import { render, screen } from '@testing-library/react';
import Main from './Main';

jest.mock('../ui/Container', () => () => <div>Container</div>)
describe('Main', () => {
    it('should render component', () => {
        const {container} = render(<Main />);
        expect(container).toMatchSnapshot();
    });
});
