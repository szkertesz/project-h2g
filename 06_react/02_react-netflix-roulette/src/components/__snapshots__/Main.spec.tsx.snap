// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render component 1`] = `
<div>
  <main>
    <div>
      Container
    </div>
  </main>
</div>
`;
